source/by-role/system-admin/upgrade/upgrading-search.rst

```diff
@@ -79,6 +84,8 @@ Here are the steps to  upgrade your CrafterCMS  3.1.x install:
 ---------------------------------------
 Upgrading 4.0.x -> 4.1.0 (from ES 7.15)
 ---------------------------------------
+When upgrading from 4.0.x (running ES 7) the indices are not compatible at all, so the content needs to be reprocessed and indices rebuilt completely. The rebuilding of the indices is handled by the post-upgrade script.
```

Comment:
Should this explicitly say `upgrade-search.sh` should not be executed when upgrading 4.0.x?